### PR TITLE
Added new doc string to clarify how to retain roll and pitch

### PR DESCRIPTION
**Pull request by MaxLarsson2**

### CODE REVIEW

File: src/pydata_tools/src/pyall_data.cpp [hunk 1]
```diff
@@ -22,7 +22,7 @@ using namespace all_data;
 namespace py = pybind11;
 
 PYBIND11_MODULE(all_data, m) {
-    m.doc() = "Basic utilities for working with the .all file format"; // optional module docstring
```
Comment (nilsbore) [inline src/pydata_tools/src/pyall_data.cpp]: Why was this removed? Looks like it should be there?

**Comment by nilsbore** [comment]
@MaxLarsson2 Thanks for the PR! Looks good except for the comment. Fix or motivate and I will merge the PR.

File: src/pydata_tools/src/pyall_data.cpp (hunk 1 above)
Comment (MaxLarsson2) [reply src/pydata_tools/src/pyall_data.cpp]: I moved the commented line "Basic utilities for working with the .all file format" to the .rst file, so that the same text should still be present, but appear on the top of the page just like before, instead of after "Important note:". I think that makes more sense from a design perspective.

Comment (nilsbore) [reply src/pydata_tools/src/pyall_data.cpp]: That makes sense. However, the doc string is still needed to get some documentation for the module for autocompletion when in a python evironment. So I would still advocate for keeping it in some form. It's fine for the line to appear in both places.

Comment (MaxLarsson2) [reply src/pydata_tools/src/pyall_data.cpp]: Okay, that is a good point I did not think of. I still feel that it is unnecessary that it appears twice  with just one short line between. So how about I re-add the doc string and remove the phrase from the .rst?
